.github/workflows/prerelease-comment.yml

```diff
@@ -0,0 +1,52 @@
+name: Write Beta Release comment
+
+on:
+  workflow_run:
+    workflows: ["CI"]
+    types:
+      - completed
+
+jobs:
+  comment:
+    if: |
+      github.repository_owner == 't3-oss' &&
+      ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
will this comment on PRs even if a new version hasn't been published?

Reply:
This would run after the pre-release workflow, I assumed if we reached here it would always have a new version. We coud explicitly fail the previous step to make sure to never run the comment if there is not a new version

Reply:
At the top it says runs after completed CI?

Reply:
That should not be there, fixing

Reply:
Ah great 👍

Reply:
That was the previous step in my mock CI